include/nccl_ofi_api.h

```diff
@@ -67,4 +70,83 @@ ncclResult_t nccl_net_ofi_iwrite_inline_v5(void* sComm, void* src, size_t size,
 ncclResult_t nccl_net_ofi_iread_v5(void* rComm, void* dest, size_t size, void* mhandle,
 				   uint64_t src, uint64_t mr_key, void** req);
 
+/* Note: this is a global variable, set based on the parameter,
+   ofi_nccl_abort_on_error(). */
+extern bool abort_on_error;
```

Comment:
This should not be a global variable.  If you need a local shadow for performance, fine, but everyone should use the parameter itself rather than make a bunch of new globals.